File: yangxj96-common/src/main/java/com/yangxj96/saas/common/stream/StreamModel.java
package com.yangxj96.saas.common.stream;

import lombok.Data;
import lombok.ToString;

/**
 * 流消息格式
 *
 * @author yangxj96
 * @version 1.0.0
 * @date 2023/4/20 14:52
 */
@Data
@ToString
public class StreamModel {

    private String type;

    private Object message;
}

File: yangxj96-bean/src/main/java/io/github/yangxj96/stream/StreamModel.kt
/*
 *  Copyright (c) 2021 - 2023
 *  作者：杨新杰(Jack Young)
 *  邮箱：yangxj96@gmail.com
 *  日期：2023-04-20 14:52:53
 *  Copyright (c) 2021 - 2023
 */

package io.github.yangxj96.stream;

import lombok.*;

/**
 * 流消息格式
 *
 * @author yangxj96
 * @version 1.0.0
 * @date 2023/4/20 14:52
 */
@Data
@ToString
@EqualsAndHashCode
@AllArgsConstructor
@NoArgsConstructor
public class StreamModel {

    private String type;

    private Object message;

}
